src/kitovu/gui/mainwindow.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, parent: QWidget = None) -> None:
 
         self._conf_screen.close_requested.connect(
             lambda: self.setCurrentWidget(self._start_screen))
-        self._sync_screen.close_requested.connect(
+        self._sync_screen.close_requested.connect(  # pragma: no branch
```

Comment:
Due to https://bitbucket.org/ned/coveragepy/issues/90/lambda-expression-confuses-branch